### ISSUE TITLE
LARAVEL 8 ISSUE 

### ISSUE THREAD

**Issue by salmanrajz** [asantibanez/livewire-calendar#7]
Return value of App\Http\Livewire\AppointmentsCalendar::events() must be an instance of Illuminate\Database\Eloquent\Collection, instance of Illuminate\Support\Collection returned (View: /Applications/MAMP/htdocs/todo/resources/views/vendor/jetstream/components/welcome.blade.php) (View: /Applications/MAMP/htdocs/todo/resources/views/vendor/jetstream/components/welcome.blade.php)

**Comment by germanio7** [comment]
the solution #6.

**Comment by germanio7** [comment]
try this

```
use Carbon\Carbon;
use Illuminate\Support\Collection;
use Asantibanez\LivewireCalendar\LivewireCalendar;

class Index extends LivewireCalendar
{
    public function events(): Collection
    {
        return collect([
            [
                'id' => 1,
                'title' => 'Breakfast',
                'description' => 'Pancakes! 🥞',
                'date' => Carbon::today(),
            ],
            [
                'id' => 2,
                'title' => 'Meeting with Pamela',
                'description' => 'Work stuff',
                'date' => Carbon::tomorrow(),
            ],
        ]);
    }
}
```

**Comment by salmanrajz** [comment]
after installing I need to change the whole laravel components location, is there any way to fix this?

**Comment by germanio7** [comment]
I did not change location, I changed this @ #6

**Comment by salmanrajz** [comment]
okay, after installation I'm getting this error.

Livewire\Exceptions\ComponentNotFoundException
Unable to find component: [navigation-dropdown]

**Comment by germanio7** [comment]
to fix that, try to change this:

/vendor/asantibenz/livewire-calendar/src/LivewireCalendarServicesProvider.php

```
<?php

namespace Asantibanez\LivewireCalendar;

use Illuminate\Support\Facades\Blade;
use Illuminate\Support\ServiceProvider;

class LivewireCalendarServiceProvider extends ServiceProvider
{
    /**
     * Bootstrap the application services.
     */
    public function boot()
    {
        $this->loadViewsFrom(__DIR__ . '/../resources/views', 'livewire-calendar');

        if ($this->app->runningInConsole()) {
            $this->publishes([
                __DIR__ . '/../resources/views' => $this->app->resourcePath('views/vendor/livewire-calendar'),
            ], 'livewire-calendar');
        }

        Blade::directive('livewireCalendarScripts', function () {
            return <<<'HTML'
            <script>
                function onLivewireCalendarEventDragStart(event, eventId) {
                    event.dataTransfer.setData('id', eventId);
                }

                function onLivewireCalendarEventDragEnter(event, componentId, dateString, dragAndDropClasses) {
                    event.stopPropagation();
                    event.preventDefault();

                    let element = document.getElementById(`${componentId}-${dateString}`);
                    element.className = element.className + ` ${dragAndDropClasses} `;
                }

                function onLivewireCalendarEventDragLeave(event, componentId, dateString, dragAndDropClasses) {
                    event.stopPropagation();
                    event.preventDefault();

                    let element = document.getElementById(`${componentId}-${dateString}`);
                    element.className = element.className.replace(dragAndDropClasses, '');
                }

                function onLivewireCalendarEventDragOver(event) {
                    event.stopPropagation();
                    event.preventDefault();
                }

                function onLivewireCalendarEventDrop(event, componentId, dateString, year, month, day, dragAndDropClasses) {
                    event.stopPropagation();
                    event.preventDefault();

                    let element = document.getElementById(`${componentId}-${dateString}`);
                    element.className = element.className.replace(dragAndDropClasses, '');

                    const eventId = event.dataTransfer.getData('id');

                    window.Livewire.find(componentId).call('onEventDropped', eventId, year, month, day);
                }
            </script>
HTML;
        });
    }
}

```

**Comment by asantibanez** [comment]
Fixed in 2.1.0.

Thanks for your interest!
